modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm

```diff
@@ -3379,7 +3381,7 @@ sub core_pipeline_analyses {
                 'methods'   => { 'ENSEMBL_PARALOGUES' => 1 },
             },
             -flow_into  => {
-                1 => { 'find_homology_id_range' => { 'mlss_id' => '#mlss_id#', 'high_conf_expected' => '0' } },
+                1 => { 'import_homology_table' => { 'mlss_id' => '#mlss_id#', 'high_conf_expected' => '0' } },
```

Comment:
Why this ?

Reply:
Because `find_homology_id_range` doesn't appear to exist anymore due to the new id generator and this was missed